jetty-gcloud/jetty-gcloud-session-manager/src/main/java/org/eclipse/jetty/gcloud/session/GCloudSessionDataStoreFactory.java

```diff
@@ -22,9 +22,9 @@
  */
 public class GCloudSessionDataStoreFactory extends AbstractSessionDataStoreFactory
 {
-    private String _namespace;
-    private int _maxRetries;
-    private int _backoffMs;
+    private String _namespace = GCloudSessionDataStore.DEFAULT_NAMESPACE;
+    private int _maxRetries = GCloudSessionDataStore.DEFAULT_MAX_RETRIES;
+    private int _backoffMs = GCloudSessionDataStore.DEFAULT_BACKOFF_MS;
```

Comment:
These 3 are embedded mode default changes.
These used to be (in embedded mode) null, 0, 0.

Reply:
See above